### PR TITLE
KTLN-266: Testing Lambdas with Mockito

### PR DESCRIPTION
**Pull request by thejoeflow**

### CODE REVIEW

File: kotlin-testing/README.md [hunk 1]
```diff
@@ -7,3 +7,4 @@ This module contains articles about testing in Kotlin
 - [Kotlin with Mockito](https://www.baeldung.com/kotlin/mockito)
 - [Writing Specifications with Kotlin and Spek](https://www.baeldung.com/kotlin/kotlin-spek)
 - [MockK: A Mocking Library for Kotlin](https://www.baeldung.com/kotlin/mockk)
+- [Testing Higher-Order Functions with Mockito](https://baeldung.com/kotlin/testing-higher-order-functions-with-mockito)
```
Comment (ashleyfrieze) [inline kotlin-testing/README.md:10]: We don't put these in at this stage, please remove it.

File: kotlin-testing/src/main/kotlin/com/baeldung/argumentcaptor/OrderService.kt [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        if (confirmRequired) {
+            messageDispatcher.sendMessage("Order Created") { success ->
+                if (success) {
+                    orderRepository.updateOrderConfirmation(orderId, true);
+                }
+                else {
+                    throw IllegalStateException("Failed to send confirmation message!")
+                }}
+        }
+        else {
```
Comment (ashleyfrieze) [inline kotlin-testing/src/main/kotlin/com/baeldung/argumentcaptor/OrderService.kt:21]: Is this spacing of `else` common for Kotlin?

**Review by ashleyfrieze — CHANGES_REQUESTED** [review]

File: kotlin-testing/README.md (hunk 1 above)
Comment (ashleyfrieze) [reply kotlin-testing/README.md:10]: Quick reminder on this one...

**Review by ashleyfrieze — CHANGES_REQUESTED** [review]

File: kotlin-testing/README.md (hunk 1 above)
Comment (thejoeflow) [reply kotlin-testing/README.md]: Removed

File: kotlin-testing/src/main/kotlin/com/baeldung/argumentcaptor/OrderService.kt (hunk 2 above)
Comment (thejoeflow) [reply kotlin-testing/src/main/kotlin/com/baeldung/argumentcaptor/OrderService.kt]: Changed it to be consistent with Kotlin standard

**Pull request closed without merging**